t/plugin/wolf-rbac.t

```diff
@@ -685,3 +685,55 @@ passed
             ngx.status = code
         }
     }
+
+
+
+=== TEST 36: add consumer with echo plugin
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/consumers',
+                ngx.HTTP_PUT,
+                [[{
+                    "username": "wolf_rbac_with_other_plugins",
+                    "plugins": {
+                        "wolf-rbac": {
+                            "appid": "wolf-rbac-app",
+                            "server": "http://127.0.0.1:1982"
+                        },
+                        "echo": {
+                            "body": "consumer merge echo plugins\n"
```

Comment:
I want to avoid having one plugin dependent on another, and I think echo is a much more basic plugin.

after a brief review of ci errors, it seems that many errors have nothing to do with this modification. when I am free, I will have a careful look

Reply:
Please fix the code lint first

Reply:
Fixed with make lint